examples/transportation/segment/road/road-with-lr-name.yaml

```diff
@@ -15,6 +15,24 @@ properties:
   subType: road
   level: 1
   connectors: [fooConnector, barConnector] # Topology: To discuss further.
+  extNames:
```

Comment:
@vcschapp Is this right?

The old way had a single LR segment with the names beneath it. But this new way the "at" needs to be duplicated for each name that applies over the same sub-segment. I think that's what we agreed to but just double-checking